packages/golden-layout/src/controls/Header.ts

```diff
@@ -654,13 +660,14 @@ export default class Header extends EventEmitter {
 
   // enables synthetic keyboard navigation of the list
   _handleFilterKeydown(e: JQuery.TriggeredEvent) {
-    if (this.dropdownKeyIndex === -1) return;
-
     if (e.key === 'Escape') {
       this._hideAdditionalTabsDropdown();
       return;
     }
 
+    //escape should always close, even if list is empty
```

Comment:
Comment should have a space after `//` to be consistent with other comments.

That being said, I think comment here could be improved. This comment should should be before the escape key check above, then you could have a comment here saying something like `// a negative dropdownKeyIndex means the list is empty` (if that's the case, I'm inferring that from this comment saying something about the list being empty though I don't see why the list could have items in it but the dropdownKeyIndex isn't set for some reason).

Reply:
list can be empty when it's filtered to something non-matching.